source/community/groups/auth-tsg/index.md

```diff
@@ -35,6 +36,7 @@ The IIIF Authentication API is an interaction pattern for managing access contro
 [zoom-link]: https://stanford.zoom.us/j/829945399?pwd=u8KPNHGHxJk
 [iiif-calendar]: {{ site.url }}{{ site.baseurl }}/community/groups/
 [international-zoom]: https://zoom.us/zoomconference
+[auth-charter]: {{ site.url }}{{ site.baseurl }}/community/groups/auth-tsg/charter
```

Comment:
I think the issue is as simple as a missing / on the end of the URL so it should be:

```
[auth-charter]: {{ site.url }}{{ site.baseurl }}/community/groups/auth-tsg/charter/
```
The md file is community/groups/auth-tsg/charter.md and I believe these always link to a directory so its community/groups/auth-tsg/charter/

Reply:
I added that but it's still saying that it doesn't exist. It's strange because when I run it locally the link works fine.